tdrs-frontend/docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
     tty: true        # docker run -t
     build:
       context: .
-      dockerfile: Dockerfile.dev
+      target: nginx
```

Comment:
The default compose file (used in CircleCI ZAP scanner) needs to target the nginx stage in order to closely simulate the production environment.